src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestActionMenu.tsx

```diff
@@ -0,0 +1,189 @@
+import CheckBoxIcon from '@mui/icons-material/CheckBox';
+import CheckBoxOutlineBlankIcon from '@mui/icons-material/CheckBoxOutlineBlank';
+import PostAddIcon from '@mui/icons-material/PostAdd';
+import CreateNewFolderIcon from '@mui/icons-material/CreateNewFolder';
+import { useHistory } from 'react-router-dom';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import { NERButton } from '../../components/NERButton';
+import { useRequestCRReview } from '../../hooks/change-requests.hooks';
+import { useSingleProject } from '../../hooks/projects.hooks';
+import { useToast } from '../../hooks/toasts.hooks';
+import { useAllUsers, useCurrentUser } from '../../hooks/users.hooks';
+import { projectWbsPipe } from '../../utils/pipes';
+import { routes } from '../../utils/routes';
+import { userToAutocompleteOption } from '../../utils/task.utils';
+import ErrorPage from '../ErrorPage';
+import { ReactElement } from 'react';
+
+interface ChangeRequestActionMenuProps {
+  isUserAllowedToReview: boolean;
+  isUserAllowedToImplement: boolean;
+  isUserAllowedToDelete: boolean;
+  changeRequest: ChangeRequest;
+  handleReviewOpen: () => void;
+  handleDeleteOpen: () => void;
+  handleReviewerIdsSelect: (Ids: number[]) => void;
+  reviewerIds: number[];
+}
+
+const ChangeRequestActionMenu: React.FC<ChangeRequestActionMenuProps> = ({
+  isUserAllowedToReview,
+  isUserAllowedToImplement,
+  isUserAllowedToDelete,
+  changeRequest,
+  handleReviewOpen,
+  handleDeleteOpen,
+  handleReviewerIdsSelect,
+  reviewerIds
+}: ChangeRequestActionMenuProps) => {
+  const {
+    data: project,
+    isLoading,
+    isError,
+    error
+  } = useSingleProject({
+    carNumber: changeRequest.wbsNum.carNumber,
+    projectNumber: changeRequest.wbsNum.projectNumber,
+    workPackageNumber: 0
+  });
+
+  const { data: users, isLoading: isLoadingAllUsers, isError: isErrorAllUsers, error: errorAllUsers } = useAllUsers();
+  const { mutateAsync: requestCRReview } = useRequestCRReview(changeRequest.crId.toString());
+  const toast = useToast();
+  const currentUser = useCurrentUser();
+  const history = useHistory();
+  if (isError) return <ErrorPage message={error?.message} />;
+  if (isErrorAllUsers) return <ErrorPage message={errorAllUsers?.message} />;
+  if (!project || isLoading || isLoadingAllUsers || !users) return <LoadingIndicator />;
+
+  const handleRequestReviewerClick = async () => {
+    if (reviewerIds.length === 0) {
+      toast.error('Must select at least one reviewer to request review from');
+      return;
+    }
+    try {
+      await requestCRReview({ userIds: reviewerIds });
+    } catch (e) {
+      if (e instanceof Error) {
+        toast.error(e.message);
+      }
+    }
+  };
+
+  const isRequestAllowed = changeRequest.submitter.userId === currentUser.userId;
+
+  const unreviewedActionsDropdown = (): ReactElement => {
+    return (
+      <div>
+        <div style={{ marginTop: '10px' }}></div>
```

Comment:
Just put the actions menu inside the div instead of an empty and then delete the other one